exit_db/src/models.rs

```diff
@@ -8,6 +8,7 @@ pub struct Client {
     pub wg_port: i32,
     pub eth_address: String,
     pub internal_ip: String,
+    pub global_ipv6: String,
```

Comment:
maybe internet_ipv6?  or just a comment indicating it's an internet routable ipv6 address assigned to this router.

Reply:
also this is a subnet, so it should include client subnet size as part of the string encoding. This makes the 'pick a non-overlapping subnet to assign to a new client' problem extremely hard becuase what if subnet assignments changed since first startup? We could maybe ignore that case and just say 'all client subnet sizes are always the same all the time' and put a value in the config.